docs/typescript.md

```diff
@@ -201,15 +205,70 @@ Advanced Service currently do not have TypeScript autocompletion.
 
 ### TypeScript Support
 
-Currently, `clasp` supports [`typescript@2.9.2`](https://www.npmjs.com/package/typescript/v/2.9.2). If there is a feature in a newer  TypeScript version that you'd like to support, or some experimental flag you'd like enabled, please file a bug.
+Currently, `clasp` supports [`typescript@3.3.3`](https://www.npmjs.com/package/typescript/v/3.3.3). If there is a feature in a newer TypeScript version that you'd like to support, or some experimental flag you'd like enabled, please open an issue.
+
+### Modules, exports and imports
+
+Currently, Google Apps Script does not support modules, hence the typical `export`/`import` patern cannot be used and the example below will fail:
```

Comment:
Apps Script does not support ES Modules. Hence, the typical `export`/`import` pattern cannot be used and the example below will fail: